packages/stateless/components/dao/tabs/AppsTab.tsx

```diff
@@ -63,25 +63,27 @@ const InnerAppsTab = ({
 }: InnerAppsTabProps) => {
   const { t } = useTranslation()
   const [iframe, setIframe] = useState<HTMLIFrameElement | null>(null)
+  const [inputUrl, setInputUrl] = useState<string>(url)
 
-  const go = (destUrl = url) => {
-    setUrl(destUrl)
-    if (iframe) {
-      iframe.src = destUrl
-    }
+  const go = (url: string) => {
+    setUrl(url)
   }
 
-  // On first iframe mount, go to url if valid already.
+  // On first iframe mount, go to url
   useEffect(() => {
     try {
-      if (iframe && url && new URL(url).href) {
+      if (iframe) {
```

Comment:
```suggestion
      if (iframe && url && new URL(url).href) {
```

i think we can leave these url validation checks in